### PR TITLE
Git - Improve git fetch command for repositories with multiple remotes

### PR DESCRIPTION
**Pull request by lszomoru**
For repositories that have more than one remote, we will display a quick pick to choose one/all remote(s) to fetch:

image

### CODE REVIEW

**Comment by lszomoru** [comment]
@joaomoreno, here are some UI tweaks based on our conversation:

(1) no icons

image

(2) icons for all

image

(3) icon only for the command

image

**Comment by joaomoreno** [comment]
(2) looks 🔥

File: extensions/git/src/commands.ts [hunk 1]
```diff
@@ -2239,7 +2261,49 @@ export class CommandCenter {
...
+		quickpick.show();
+		const remoteItem = await new Promise<RemoteItem | FetchAllRemotesItem | undefined>(resolve => {
+			let isAccepted = false;
+
+			quickpick.onDidAccept(() => {
+				isAccepted = true;
+				resolve(quickpick.activeItems[0] as RemoteItem | FetchAllRemotesItem);
+			});
+			quickpick.onDidHide(() => {
+				if (!isAccepted) {
```
Comment (joaomoreno) [inline extensions/git/src/commands.ts:2295]: Is this `isAccepted` variable really necessary? Calling `resolve` twice is a no-op.

Comment (lszomoru) [reply extensions/git/src/commands.ts]: Makes sense. I have cleaned that up.

**Review by joaomoreno — APPROVED** [review]

**Pull request merged**